src/components/common/ContentLoader.tsx

```diff
@@ -37,7 +35,7 @@ const ContentLoader = ({ fullPage = false }: ContentLoaderProps) => {
   }
 
   return (
-    <Loader fullPage={fullPage}>
+    <Loader style={{ height: fullPage ? 'calc(100vh - 24rem)' : '3rem' }}>
```

Comment:
What was the issue with these styles inside the `Loader` styled component?

Reply:
For some reason got console warnings on having the `fullPage` prop in the tag. This was a quick fix to get rid of those. Usually custom props for styled components work fine but I couldn't immediately see what was different with this one.

Reply:
All custom props for styled components should be prefixed with a `$`, otherwise they'll be passed on as-is to the underlying DOM element. `fullPage` is not a prop that `<div>` supports, so the name of the prop should be `$fullPage` instead.